test/functional/lua/fs_spec.lua

```diff
@@ -82,7 +82,11 @@ describe('vim.fs', function()
 
   describe('normalize()', function()
     it('works with backward slashes', function()
-      eq('C:/Users/jdoe', exec_lua [[ return vim.fs.normalize('C:\\Users\\jdoe') ]])
+        local expected = [[C:\Users\jdoe]]
+        if iswin() then
+          expected = 'C:/Users/jdoe'
+        end
+      eq(expected, exec_lua [[ return vim.fs.normalize('C:\\Users\\jdoe') ]])
```

Comment:
normalize() should also collapse redundant consecutive slashes to a single slash, e.g. 

```
foo///bar/ => foo/bar
foo//\\bar/ => foo/bar
```

Do we have test cases for this?